### PR TITLE
Feat/prometheus server update

### PR DESCRIPTION
**Pull request by sobrase**
Updated prometheus_server role, in order to only install snmp_exporter and ipmi_exporter if necessary.
Also added inventory variables for alertmanager.yml (should be added into defaults.yml ?) , and added restart of prometheus service each time the role is executed, in order to take into account the changes to prometheus.yml. The variables start_services and enable_services for prometheus_server and prometheus_client which are required should also be placed into default.yml?

### CODE REVIEW

**Comment by btravouillon** [comment]
Hi @sobrase ,

> Updated prometheus_server role, in order to only install snmp_exporter and ipmi_exporter if necessary.

Can we install those packages with the role prometheus_client? Or is it mandatory to collocate these 2 exporters with the prometheus service?

**Comment by oxedions** [comment]
Dear @sobrase 

I will have a deep look into this end of next week.

File: roles/addons/prometheus_client/readme.rst [hunk 1]
```diff
@@ -11,6 +11,33 @@ Instructions
...
+.. code-block:: yaml
+
+monitoring:
+
+  exporters:
+    node_exporter:
+      package: node_exporter
+      service: node_exporter
+      port: 9100
+    ha_cluster_exporter:
```
Comment (btravouillon) [inline roles/addons/prometheus_client/readme.rst:26]: This exporter is not provided with bluebanquise.

**Comment by btravouillon** [comment]
Can you use this PR to fix the indentation in prometheus doc at https://github.com/bluebanquise/bluebanquise/blame/eea7e83a42fd7f25fc73dee6a1ef774500db2c09/resources/documentation/monitoring/prometheus.rst#L36-L41

**Comment by oxedions** [comment]
We need to review the snmp and ipmi architecture for 1.4 version. This PR is to be left open until investigated.

**Comment by oxedions** [comment]
Closing for now.

**Pull request closed without merging**